### PR TITLE
fix: lets just log a warning if the alpha commands can't be found

### PR DESCRIPTION
**Pull request by jstrachan**
this can happen if github is having issues or there are firewall problems accessing github

this avoids failing on all jx commands which is pretty fatal

Signed-off-by: James Strachan <james.strachan@gmail.com>

### CODE REVIEW

File: pkg/cmd/alpha/alpha.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func NewCmdAlpha(commonOpts *opts.CommonOptions) (*cobra.Command, error) {
 
 	plugins, err := options.getPluginSpecs()
 	if err != nil {
-		return cmd, err
+		log.Logger().Warnf("failed to discover alpha commands from github: %s", err.Error())
```
Comment (garethjevans) [inline pkg/cmd/alpha/alpha.go]: should we return at this point? I've not looked at the code so am unsure at what `plugins` will be

Comment (garethjevans) [reply pkg/cmd/alpha/alpha.go]: it looks like it could be nil

Comment (jstrachan) [reply pkg/cmd/alpha/alpha.go]: its fine for it to be nil as iterating over a nil list is fine in go

e.g.
https://play.golang.org/p/igI2_8v9StC

if its nil it just doesn't add any sub commands

Comment (jstrachan) [reply pkg/cmd/alpha/alpha.go]: @garethjevans any chance I can convince you to /lgtm again? :)

**Pull request merged**